addons/barcodes/__openerp__.py

```diff
@@ -1,23 +1,29 @@
 {
     'name': 'Barcodes',
-    'version': '1.0',
-    'category': '',
-    'sequence': 6,
-    'summary': 'Barcode Nomenclatures Setup',
+    'version': '2.0',
+    'category': 'Hidden/Dependency',
+    'summary': 'Barcodes Scanning and Parsing',
     'description': """
+This module adds support for barcode scanning and parsing.
 
-=======================
+Scanning
+--------
+Use an USB scanner (that mimics keyboard inputs) in order to work with barcodes in Odoo.
+The scanner must be configured to use no prefix and a carriage return or tab as suffix.
+The delay between each character input must be less than or equal to 50 milliseconds.
+This is how most barcode scanners will work out of the box.
+However, make sure the scanner uses the same keyboard layout as the device it's plugged in.
```

Comment:
Depending on the use case it might be easier to change the device's keyboard layout (card readers) or the scanner's keyboard layout (barcode scanners whose manual you haven't lost). I updated that paragraph accordingly